AnnoyancesFilter/sections/css_extended.txt

```diff
@@ -96,6 +96,7 @@ facebook.com,facebookcorewwwi.onion#%#(function() { var observer=new MutationObs
 afisha.ru##.page__footer > section:has(> .page__footer-apps)
 eva.ru##div:has(> div > .subscribe-form)
 grafixfather.com#?#.td-post-content > pre:contains(For More Secret Updates Join Us)
+tvplay.tv3.lt#?##__next > div > div:has(> div:has(> a) + button)
```

Comment:
A bit dangerous rule since no cookie related element is included.
tvplay.tv3.lt#?##__next > div[class] > div[class]:has(> div:contains(privatumo))

Reply:
What about to not add this rule? On my end this selector does not find the necessary element.

Reply:
Cookie message at top:

<img width="943" alt="tvplay" src="https://user-images.githubusercontent.com/34266271/60688765-7e7f8180-9eb7-11e9-9d91-165ab0ba4066.png">